### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by brooklinr9**
I basically followed the Homework Workflow instruction in creating change to hw01 repository: I forked and cloned the original repository using Github Desktop; then I used RStudio to modify the README.md documents. In modifying the README.md document, I find the R Markdown tutorial(https://bookdown.org/yihui/rmarkdown/#)especially helpful.

One problem I had in modifying the README.md document is creating a sub-list. I tried several formats but still was not able to do so.

### CODE REVIEW

**Comment by cysakamoto** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |     x         |            |
| **Coding strategy**         |           |              |    x        |
| **Presentation**            |           |      x        |            |
| **Achievement, creativity** |           |     x         |            |
| **Ease of access**          |     x      |              |            |

Hi Ruixi, 

Good use of markdown elements, such as bold, headings, and bullets, but your image didn't show up and there is no link to your Linkedin. Let me know if you still have issues and need to talk about it. I'm in sociology too, and it's nice to know you're taking this course! 

Cristina

File: README.md [hunk 1]
```diff
@@ -1,2 +1,22 @@
...
+## Biography ##
+***Basic Information***  
+
+Name:Ruixi(Ray) Mao  
+
+*  Current Status: third year grad student in Sociology 
+  +  Academic Interest: Economic Sociology; 
+  +  Current Research: Capitalism; Entrepreneurship  
+  
+[here's my linkedin profile]()
```
Comment (smiklin) [inline README.md:11]: You'd want to include the URL  in the brackets here for the link to work :-)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,22 @@
...
+
+***Other Interests***  
+
+Travel: I recently went ot Tibet:
+
+![](IMG_1192.jpg)
```
Comment (smiklin) [inline README.md:17]: being that your image is in the same folder as the .md file, I'm wondering if the only thing you'd need for this to work is a `/`, so ``

File: README.md [hunk 3]
```diff
@@ -1,2 +1,22 @@
-# hw01
-Repository for homework 01
+# **Homework 1**
+## Biography ##
+***Basic Information***  
+
+Name:Ruixi(Ray) Mao  
+
+*  Current Status: third year grad student in Sociology
```
Comment (smiklin) [inline README.md:7]: I think, if you want this to be a sub-list, you might want to add another tab before sub-list items

**Review by smiklin — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |       v       |            |
| **Coding strategy**         |           |              |     v       |
| **Presentation**            |           |        v      |            |
| **Achievement, creativity** |           |       v       |            |
| **Ease of access**          |      v     |              |            |

Remarks:

I've personally found it really helpful to run a code a few times to make sure that everything works, and if it doesn't, I copy-paste an example I found online and compare--> I've found the formatting to be quite sensitive to spaces and indentations. Nice pic!

File: README.md (hunk 3 above)
Comment (brooklinr9) [reply README.md:7]: I've got things fixed according to your advices. thx!

**Pull request closed without merging**